Agrega la vista de generacion de Informe de Migracion de Pedidos.

arsenioFerreira opened the pull request:
- Permite generar archivos en formato PDF y XLS.

arsenioFerreira committed:
Agrega la vista de generacion de Informe de Migracion de Pedidos.
- Permite generar archivos en formato PDF y XLS.

arsenioFerreira committed:
- Agregacion de converters en la vista de MigracionPedido.
- Correccion de l reporte generado.

lokolte commented on SisVenLog/SisVenLog-war/src/java/util/LlamarReportes.java:
```diff
@@ -60,7 +60,7 @@
         try {
             Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
-            conexion = DriverManager.getConnection("jdbc:sqlserver://localhost;databaseName=VenlogDB", "sa", "venlog2018CC");
+            conexion = DriverManager.getConnection("jdbc:sqlserver://localhost;databaseName=VenlogDB", "venlog", "venlog2018CC");
```
Evitar cambiar datos de conexiones a base de datos, solo para usos locales, no commitear al servidor.

lokolte commented on SisVenLog/SisVenLog-war/src/java/bean/listados/LiMigracionPedidos.java:
```diff
@@ -0,0 +1,261 @@
+            };
+            
+            
+            String sql = ""
```
Esto se puede procesar en un facade asociado a este bean y evitar la mezcla de codigo sql y logica del bean en una misma clase.

lokolte commented on SisVenLog/SisVenLog-war/src/java/bean/listados/LiMigracionPedidos.java:
```diff
@@ -0,0 +1,261 @@
+            
+            sql += " ORDER BY cod_vendedor, nroped";
+
+            lista = excelFacade.listarParaExcel(sql);
```
con el retorno del sql del facade se llamaria a este reporte xls.

arsenioFerreira committed:
- Refactor de la clase Bean moviendo la logica de generacion de sql
a su clase Facade.
- Correccion en el usuario de conexion para la llamada de Reportes.
- Correcciones varias en el reporte generado.

arsenioFerreira committed:
- Correccion en el combo de canales de venta.
- Correccion en el reporte con pruebas realizadas con datos reales.
- Agregacion de la descripcion del canal.
